include/nccl_ofi.h

```diff
@@ -69,6 +69,12 @@ extern "C" {
 /* Flush read size (bytes) */
 #define NCCL_OFI_FLUSH_SIZE	4
 
+/* Maxiumum registration key value when not using FI_MR_PROV_KEY */
+#define NCCL_OFI_MAX_MR_KEY	256
```

Comment:
How has this number been decided?

Reply:
This asks the provider to support an 8 bit MR key (specified by NCCL_OFI_MR_KEY_BYTES a few lines down), which is the minimum. I'll double check with the NCCL team that this is a sufficient baseline. I could make it an env var if we feel that users might need to adjust this value for some workloads.

Reply:
This should be calculated off of `NCCL_OFI_MR_KEY_BYTES` as follows (will make this change):
```
#define NCCL_OFI_MAX_MR_KEY>----(1 << (NCCL_OFI_MR_KEY_BYTES * 8))
```